src/aws_encryption_sdk/identifiers.py

```diff
@@ -271,9 +273,27 @@ class WrappingAlgorithm(Enum):
     :type padding_mgf:
     """
 
-    AES_128_GCM_IV12_TAG16_NO_PADDING = (EncryptionType.SYMMETRIC, Algorithm.AES_128_GCM_IV12_TAG16, None, None, None)
-    AES_192_GCM_IV12_TAG16_NO_PADDING = (EncryptionType.SYMMETRIC, Algorithm.AES_192_GCM_IV12_TAG16, None, None, None)
-    AES_256_GCM_IV12_TAG16_NO_PADDING = (EncryptionType.SYMMETRIC, Algorithm.AES_256_GCM_IV12_TAG16, None, None, None)
+    AES_128_GCM_IV12_TAG16_NO_PADDING = (
```

Comment:
this is a function of tox -re autoformat (and python black) I believe.

Reply:
Auto-formatting does this because some lines are too long, but some aren't.  Making them all one line will fail flake8.  They do pass if you make them all the new format, however.  So the options are either to add some variables to shorten the text and make them all one line or make them all the new format.  I think the latter is more readable, but both are valid options.  I've committed changes to standardize to the second option tentatively.